ebuild-writing/misc-files/patches/text.xml

```diff
@@ -112,6 +112,42 @@ from the <c>vim</c> patch tarball:
 </body>
 </section>
 
+<section>
+<title>Conditional Patching</title>
```

Comment:
```suggestion
<title>Conditional patching</title>
```
@maffblaster is going to change all existing titles to sentence case soon, so I guess newly added ones should also follow that.

Reply:
Done.